CharlieBackend.Business/Services/AccountService.cs

```diff
@@ -105,11 +105,11 @@ public async Task<Account> GetAccountCredentialsByIdAsync(long id)
             return null;
         }
 
-        public async Task<IList<AccountDto>> GetAllNotAssignedAccountsAsync()
+        public async Task<Result<IList<AccountDto>>> GetAllNotAssignedAccountsAsync()
```

Comment:
Don't forget to call .ToActionResult in controller action. Otherwise, it will change schema of response body.